src/utilities/OpenWeather.tsx

```diff
@@ -0,0 +1,21 @@
+import axios from 'axios';
+
+const getTemperature = (zipcode: string) => {
+  // Returns promise in which value contains the temperature or error
+  return axios.get('http://api.openweathermap.org/data/2.5/weather',
+  {params: {
+    zip: zipcode,
+    units: 'imperial',
+    appid: process.env.REACT_APP_OPENWEATHER_API_KEY}}
+  )
+  .then(function (response) {
```

Comment:
We should utilize the new ES features and use arrow functions here. 

`.then((response) => {...`

Also in the catch section below

Reply:
Added